docs/usage/release-notes-for-major-versions.md

```diff
@@ -7,6 +7,69 @@ The most recent versions are always at the top of the page.
 This is because recent versions may revert changes made in an older version.
 You also don't have to scroll to the bottom of the page to find the latest release notes.
 
+## Version 39
+
+### Breaking changes for 39
+
+#### New tools and config for Renovate's Docker images
+
+Renovate Docker images now use Node.js v22 as base, this was Node.js v20.
+
+##### Sidecar images
+
+Renovate's "sidecar" images now:
+
+- Use Ubuntu 24.04 as base, this was 20.04
+- Set the user ID to `12021`, this was `1001`
```

Comment:
All renovate images are now having this, they are all extend the sidecar image. so move one heading up

Reply:
Also add a note that caches need to be removed or perms need to be fixed because of the userid change.